src/components/Toaster/Toast.tsx

```diff
@@ -15,7 +15,7 @@ type ToastType = 'info' | 'warn' | 'error' | 'success' | 'progress';
 
 export interface ToastProps {
   /** JSX Content of Toast */
-  content: React.ReactNode;
+  content?: React.ReactNode;
```

Comment:
rather than having it required and accepting a null, we made it optional to reduce complexity on stories and we're defaulting it to `null`.